pq-crypto/bike_r2/gf2x_mul.c

```diff
@@ -23,18 +23,16 @@
 #include "gf2x_internal.h"
```

Comment:
It looks like this whole file is being reverted to an old version. I believe Jonathan Henson found a bug in here (aside from the comment formatting) that was causing issues. Checkout this commit https://github.com/awslabs/s2n/commit/ddca318218b6b931c2dd2aae0e69a3f7c4eb6697

Reply:
The alignment issue fix is there see [Line 90](https://github.com/awslabs/s2n/pull/1527/files/45f7a2d50b63a2db2219e4015cf4ffa6763b30b1#diff-3aa6b27f57035c5c3d9704068ff9e76bR90).
When copying BIKE's code we are copying the entire BIKE's repository, and therefore we are copying its style. This explains why the comments style are different.